### ISSUE TITLE
some eol page ids to not resolve to a taxonomic classification using the eol api

### ISSUE THREAD

**Issue by jhpoelen** [jhpoelen/eol-globi-data#191]
In the name reports accessed at 16 Nov 2015 (```unmatchedTaxa.csv```, see https://github.com/jhpoelen/eol-globi-data/wiki/Taxonomy-Matching), about 1300 page ids appear. This means that while the ids failed to retrieve a full classification (or taxon path) using the EOL api. 

Examples include:
http://eol.org/pages/10193006
http://eol.org/pages/1065056
http://eol.org/pages/9138291

For completeness, a complete list is included as well as a copy of the full unmatched names report.

@millerse @jhammock @KatjaSchulz please confirm that this is expected behavior. Also, let me know what we can do to make sure that full taxonomic classification can be retrieved.

[eolPageIdsWithoutClassification.txt](https://github.com/jhpoelen/eol-globi-data/files/35635/eolPageIdsWithoutClassification.txt)

[unmatchedTaxa.csv.txt](https://github.com/jhpoelen/eol-globi-data/files/35638/unmatchedTaxa.csv.txt)

**Comment by jhammock** [comment]
Sorry, Jorrit; trying to diagnose this, and my first guess is you fell victim to some of the consequences of our deploy earlier this morning, but can you define "full classification" in this instance? A few of those IDs, at the moment, do not exist on EOL, many of them have very sparse classification, and a few have pretty reasonable ones, though I haven't found any from any of our "browsing classification providers", which might make a difference.

**Comment by jhpoelen** [comment]
Thanks for having glance at this. The ids included used the eol api over the weekend, so would not include a information retrieved using the newer version deployed earlier today. By "full classification", I mean linking a taxon to its taxonomic parents. 

What would make it easier for you to distinguish between unknown ids and ids that do not have a classification? Perhaps use a specific name status like: id link error?

**Comment by jhammock** [comment]
Also- @KatjaSchulz tells me the name providers shown on the pages I did find (eg: biolib.cz, SPIRE) may all be in our DO NOT USE names category. We don't rely on them for mapping because they are so dirty. Do you know if you are using only some of our names providers? 

Just checking: most of the names have a genus, or maybe a genus and family, above them right now on their taxon pages. That would suffice, by your definition? It's fewer layers of parents than most, I think.

How much does this result differ from what you've gotten previously? (if at all)

A flag to distinguish names vanished from EOL and names orphaned on EOL (in the unmatched taxa file, you mean?) would be lovely.

**Comment by jhpoelen** [comment]
GloBI only uses name providers exposed through the eol api. Some spot checks suggest that the "do not use" providers are not included in this.

As far as the depth or length of the taxonomic path / classification - a spotty classification is ok (e.g. missing kingdom; only family genus species). 

I don't think the results are anything new, I just happen to notice it as part of validating #187 . I figured it might be useful to document this observation.

I've documented this new mini-feature idea in issue #192.
